app/presenters/publishing_api/publication_presenter.rb

```diff
@@ -75,7 +75,20 @@ def body
 
     def documents
       return [] unless item.attachments.any?
-      Whitehall::GovspeakRenderer.new.block_attachments(item.attachments, alternative_format_email)
+      Whitehall::GovspeakRenderer.new.block_attachments(
+        attachments_for_current_locale,
+        alternative_format_email
+      )
+    end
+
+    def attachments_for_current_locale
+      attachments = item.attachments
+      locales_that_match = Array(I18n.locale.to_s)
+      locales_that_match << "" if I18n.locale == I18n.default_locale
```

Comment:
What's the purpose of pushing an empty string here? Does it match attachments with an empty locale on line 90?